tests/configurations/integration/jest.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
   collectCoverageFrom: ['<rootDir>/src/**/*.{ts}', '!**/node_modules/**', '!**/vendor/**'],
   preset: 'ts-jest',
   testEnvironment: 'node',
+  testTimeout: 100000,
```

Comment:
why set a global timer for all tests? and even if it is correct, why such a high value, we previously had 20000

Reply:
to prevent optional timing issues, all the tests takes 60 seconds approximately even with this definitions